.travis.yml

```diff
@@ -66,6 +66,19 @@ jobs:
     - SENTRY_DSN=https://e012761929d24baa96ca82772f60b395@sentry.io/1203154
     - LOGROCKET_SLUG=et0uhu/streamr-frontend-local
     script: "$TRAVIS_BUILD_DIR/app/travis_scripts/visual-tests.sh"
+  - stage: Build docker (dev)
+    install: true
+    env:
+      - OWNER=streamr
+      - IMAGE_NAME=platform
+      - TAG=dev
+    script:
+      - npm ci
+      - npm run build
+      - docker build -t $OWNER/$IMAGE_NAME:$TAG --build-arg NPM_TOKEN=${NPM_TOKEN} .
```

Comment:
When should this stage run? It seems like this should probably only run for the `development` or `master` branch, otherwise won't PRs be constantly clobbering the 'dev' image?

One option could be to change `$TAG` to be branch-specific e.g. `$TRAVIS_BRANCH`/`$TRAVIS_PULL_REQUEST_BRANCH`?

Reply:
Changed definition so that docker is only built and pushed for `master` branch